app/views/index.njk

```diff
@@ -2,10 +2,14 @@
 
 {% set primaryNavId = 'applications' %}
 
+{% set title %}
+  Applications ({{applications.length}})
+{% endset %}
+
```

Comment:
This could be:

```
{% set title = "Applications (" + applications.length + ")" %}
```

What happens if `applications.length` is zero or undefined? At the moment, the solution would show empty brackets.

Reply:
oh yeah, will sort